sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOITHelper.java

```diff
@@ -36,4 +42,55 @@ public static String getHashForRecordCount(int recordCount, Map<Integer, String>
     }
     return hash;
   }
+
+  public static void retry(RetryFunction function, int attempts, long delay)
```

Comment:
Set default value for maxAttemps and delay ?

Reply:
Ok